lib/ProductOpener/Display.pm

```diff
@@ -6964,19 +6983,6 @@ sub display_field($$) {
 		
 		# Separate To-Do and Done Status
 		if ($field eq 'states') {
-			my $done_status = '';
-			my $to_do_status = '';
-			my @status_split = split(',', $value);
-			foreach my $val (@status_split) {
-				if ((index($val, "to-be") != -1) or (index($val, "Empty") != -1)) {
-		 			$to_do_status .=$val . ",";
-  		 		}
-		 		else {
-		 		$done_status .=$val . ",";
-		 		}
-			}
-			$to_do_status =~ s/,$//;
-			$done_status =~ s/,$//;
 			if ($to_do_status ne ""){
```

Comment:
What's in the "if ($field eq 'states') {" condition should now be moved above, (after you set the values $to_do_status and $done_status), as you already test for that condition above.

Reply:
So should I add "if ($field ne 'states') { $html .= '<p><span class="field">' . $lang_field . separator_before_colon($lc) . ":</span> $value</p>";}"
So that the status does not print again

Reply:
Yes, you can do it like that.